### PR TITLE
bounty: unify modal designs

### LINKED ISSUE #430: Unify Modal Designs
As a user I want a consistent modal across the Gitcoin app, so that I have consistent understanding and expectations on how to interact with them.

# Description
Standardize all modals/lightboxes.
Standardize how we communicate that information is being processed. Leverage Tips lightbox background but use new UI style defined in the attached.

# Definition of Done
- All modals and waiting states are using Muli Font
- Screens 3 is updated with the look and feel from screenshot 2
- A consistent message is used on how we communicate information processing for fund and issue and for sending tips (screen 1 & 2). 
- No major functional changes

# Reference Materials
Screenshot 1
screen shot 2018-03-05 at 12 15 46 pm

Screenshot 2
screen shot 2018-03-05 at 2 41 31 pm

Screenshot 3
screen shot 2018-03-13 at 5 08 02 pm

# Reviewers
@PixelantDesign

### PR DESCRIPTION
**Pull request by sethmcleod**
##### Description
WIP

This will resolve #430 

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
UI, maybe Dashboard? Not 100% sure how to answer this 😅 

##### Refers/Fixes

Fixes: #430

### CODE REVIEW

**Comment by sethmcleod** [comment]
Thanks @mbeacom! This is still a WIP at the moment; will be ready as soon as I finish implementing the new _Get Started_ modal based on this [feedback](https://github.com/gitcoinco/web/issues/430#issuecomment-379488306).

**Comment by mbeacom** [comment]
@sethmcleod Is there anything I can do to assist you in getting this across the finish line?

**Comment by sethmcleod** [comment]
Hey @mbeacom. Unfortunately I won't be able to wrap this one up; would you or anyone else be able to take over? Based on #430, I believe all that's left is to setup a new `/get-started` page that uses _yge_ assets similar to `/tip` so we can give it the same background/feel.

**Comment by thelostone-mc** [comment]
@sethmcleod You mean just this [bit](https://github.com/gitcoinco/web/issues/430#issuecomment-374774543) is left ?

Whatever you've updated till now -> it's completed / mergeable right ?
If not we could get it to a mergeable state + resolve conflicts -> I'll pick it up

File: app/assets/yge/sass/libs/_vars.scss [hunk 1]
```diff
@@ -30,11 +30,11 @@
 	$palette: (
 		bg:					#ffffff,
 		bg-alt:				#e1dfe8,
-		fg:					#414f57,
+		fg:					#000000,
```
Comment (mbeacom) [inline app/assets/yge/sass/libs/_vars.scss]: 😂

**Pull request merged**